megamek/src/megamek/common/util/BoardUtilities.java

```diff
@@ -64,34 +81,85 @@ public static Board combine(int width, int height, int sheetWidth, int sheetHeig
      * @param boards an array of the boards to be combined
      * @param isRotated Flag that determines if any of the maps are rotated
      * @param medium Sets the medium the map is in (ie., ground, atmo, space)
+     * @deprecated Use the {{@link #combine(int, int, int, int, Board[], int)}}
+     */
+    @Deprecated(since="0.50.06", forRemoval = true)
+    public static Board combine(int width, int height, int sheetWidth, int sheetHeight, Board[] boards,
+          List<Boolean> isRotated, int medium) {
+        return combine(width, height, sheetWidth, sheetHeight, boards, medium, 1);
+    }
+
+    private record PaddingArea(int startX, int startY, int columnPadding, int height){};
+
+    /**
+     * Combines one or more boards into one huge megaboard!
+     *
+     * @param width the width of each individual board, before the combine
+     * @param height the height of each individual board, before the combine
+     * @param sheetWidth how many sheets wide the combined map is
+     * @param sheetHeight how many sheets tall the combined map is
+     * @param boards an array of the boards to be combined
+     * @param medium Sets the medium the map is in (ie., ground, atmo, space)
+     * @param columnPaddingWidth Sets the padding width for the combined map
      */
     public static Board combine(int width, int height, int sheetWidth, int sheetHeight,
-                                Board[] boards, List<Boolean> isRotated, int medium) {
```

Comment:
isRotated was not used by this function, apparently it's just an old appendage that made sense sometime in the past.